database.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE IF NOT EXISTS `Database shizzle`;
+
+CREATE TABLE `table enzo `(
+	ID      INT              NOT NULL AUTO_INCREMENT,
+  	Author  VARCHAR (20)    ,
+    Title   VARCHAR (100) UNIQUE NOT NULL, 
+   	Text    VARCHAR(1000)    ,
```

Comment:
i had 2000 characters but its own preference

Reply:
why are your titles unique? does it mean if a different author creates thesame title as another author it not okay?